src/core/send.h

```diff
@@ -352,7 +352,7 @@ typedef enum QUIC_SEND_FLUSH_REASON {
     REASON_STREAM_ID_FLOW_CONTROL,
     REASON_AMP_PROTECTION,
     REASON_SCHEDULING,
-    REASON_ROUTE_COMPLETION,
+    REASON_ROUTE_COMPLETION, // TODO: unused
```

Comment:
If it's not needed, remove it.

Reply:
I made this TODO because I wasn't sure whether I can just remove the enum value and associated valuemap entry from the tracing manifest. Will it cause issues (e.g. with trace analysis tools) if a new flush reason is added later on and given the same value as this reason I'm removing? If so then instead I should mark it deprecated but not remove it.

Reply:
Since we are queueing on QuicSendFlush returning FALSE now, this is moot since we still need the queue reason